packages/labs/cli/src/lib/init/element-starter/index.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {FileTree, writeFileTree} from '@lit-labs/gen-utils/lib/file-utils.js';
+import {generateTsconfig} from './templates/tsconfig.json.js';
+import {generatePackage} from './templates/package.json.js';
+import {generateIndex} from './templates/index.html.js';
+import {generateGitignore} from './templates/gitignore.js';
+import {generateNpmignore} from './templates/npmignore.js';
+import {generateElement} from './templates/lib/element.js';
+import {CommandResult} from '../../command.js';
+import {InitCommandOptions} from '../../commands/init.js';
+import {LitCli} from '../../lit-cli.js';
+import path from 'path';
+
+export const generateLitElementStarter = async (
+  options: InitCommandOptions
+): Promise<FileTree> => {
+  const {name, lang} = options;
+  let files = {
+    ...generatePackage(name, lang),
+    ...generateIndex(name),
+    ...generateGitignore(lang),
+    ...generateNpmignore(),
+    ...generateElement(name, lang),
```

Comment:
This block makes me happy :)

my only thought is if in the future we want a utility function to ensure that we don't accidentally have file name collisions.